### ISSUE TITLE
Morpheus v5.4.5 failing

### ISSUE THREAD

**Issue by lukelloydagi** [tryfan/python-morpheus-cypher#14]
Since upgrading to Morpheus v5.4.5 I seem to be getting an error message:

```
Task Execution Failed on Attempt 1
Traceback (most recent call last):
  File "<stdin>", line 19, in <module>
  File "/var/opt/morpheus/morpheus-local/workspace/py-f522b6c9-91fa-4163-8c66-c87e78b0a02d/lib/python3.8/site-packages/morpheuscypher/__init__.py", line 128, in get
    raise Exception(data['msg'])
Exception: Looks like the server threw a gasket
```

**Comment by tryfan** [comment]
It looks like this is due to a Morpheus bug.  A fix should be released shortly in 5.4.5-2.

**Comment by tryfan** [comment]
Morpheus 5.4.5-2 was released.  That will fix this issue.  Please let me know if you are still experiencing a problem.
